rust/agama-dbus-server/src/network/model.rs

```diff
@@ -55,6 +57,13 @@ impl NetworkState {
         self.connections.iter_mut().find(|c| c.id() == id)
     }
 
+    /// Get connection by name
+    ///
+    /// * `name`: connection name
```

Comment:
Wrong name

Reply:
It is not used at the end so, removing it.